docs/sources/alerting/notifications.md

```diff
@@ -75,20 +75,22 @@ able to access the image.
 
 {{< imgbox max-width="40%" img="/img/docs/v4/slack_notification.png" caption="Alerting Slack Notification" >}}
 
-To set up slack you need to configure an incoming webhook url at slack. You can follow their guide on how
+To set up Slack, you need to configure an incoming Slack webhook URL. You can follow their guide on how
 to do that [here](https://api.slack.com/incoming-webhooks). If you want to include screenshots of the firing alerts
 in the Slack messages you have to configure either the [external image destination](#external-image-store) in Grafana,
 or a bot integration via Slack Apps. Follow Slack's guide to set up a bot integration and use the token provided
 (https://api.slack.com/bot-users), which starts with "xoxb".
 
 Setting | Description
 ---------- | -----------
-Url | Slack incoming webhook url.
+Url | Slack incoming webhook URL.
 Username | Set the username for the bot's message.
 Recipient | Allows you to override the Slack recipient.
 Icon emoji | Provide an emoji to use as the icon for the bot's message. Ex :smile:
 Icon URL | Provide a url to an image to use as the icon for the bot's message.
-Mention | make it possible to include a mention in the Slack notification sent by Grafana. Ex @here or @channel
+Mention Users | Optionally mention one or more comma separated users in the Slack notification sent by Grafana. You have to refer to users via their Slack IDs (you can get a user's ID from their Slack profile).
```

Comment:
```suggestion
Mention Users | Optionally mention one or more comma-separated Slack IDs in the Slack notification sent by Grafana. You can get a user's ID from their Slack profile.
```

Reply:
@oddlittlebird Based on my knowledge of the Slack API specs and my practical usage of it, I think the suggested wording is less technically precise. If I were to read this documentation in order to understand the semantics of the setting, I would definitely find the original wording the most helpful. The reason being that the resulting message mentions users and not IDs; the IDs (and not any kind of Slack IDs, but _user_ IDs in particular) just happen to be how users are referenced when communicating with the Slack API.

Reply:
I was mostly trying to combine the two sentences, since they seemed kind of choppy. What about Slack user IDs? If that does not work for you, then just make sure the hyphen in "comma-separated" is added and I shall be happy.

Reply:
@oddlittlebird I changed to "comma-separated already". I think I see how I can improve the sentence. The basic structure of the sentence is necessary, since while the goal is to mention Slack users, for this setting you have to provide user IDs, so you can't just refer to the latter.

Reply:
@oddlittlebird I rewrote the sentences a bit, to make it clear that the IDs of the setting must be comma-separated, and that it doesn't apply to the resulting mentions in Slack.

Reply:
Much better, thank you!